src/Entity/Form/UnsubscribeConfirmForm.php

```diff
@@ -35,13 +35,6 @@
  */
 class UnsubscribeConfirmForm extends EntityConfirmFormBase {
```

Comment:
I'm seeing the following error when I try to unsubscribe:

`Error while cancelling plan: Property "keepOriginalStartDate" has not been initialized on instance of Apigee\Edge\Api\Monetization\Entity\StandardRatePlanRevision class. Expected type: "boolean".`

Reply:
I see it when trying to subscribe.

Reply:
This will be fixed in a follow up task.